nav_msgs/OccupancyGrid map
int32 n_width
int32 n_height
float64 resolution
